src/test/java/cloud/fogbow/accs/core/datastore/services/RecordServiceTest.java

```diff
@@ -0,0 +1,156 @@
+package cloud.fogbow.accs.core.datastore.services;
+
+import cloud.fogbow.accs.constants.Messages;
+import cloud.fogbow.accs.core.datastore.orderstorage.RecordRepository;
+import cloud.fogbow.accs.core.exceptions.InvalidIntervalException;
+import cloud.fogbow.accs.core.models.AccountingUser;
+import cloud.fogbow.accs.core.models.Record;
+import cloud.fogbow.accs.core.models.UserIdentity;
+import cloud.fogbow.accs.core.models.orders.OrderState;
+import cloud.fogbow.common.models.SystemUser;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.powermock.modules.junit4.PowerMockRunnerDelegate;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.sql.Timestamp;
+import java.text.ParseException;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@PowerMockIgnore({"javax.management.*"})
```

Comment:
Try to describe what the annotation on this line does to make it clear why it is being used in the test.